lib/taskana-core/src/main/java/pro/taskana/monitor/internal/MonitorMapperSqlProvider.java

```diff
@@ -176,10 +194,16 @@ public static String getTaskIdsForSelectedItems() {
         + "</if>"
         + "</if>"
         + "<if test=\"_databaseId == 'db2'\">"
-        + "#{selectedItem.upperAgeLimit} >= (DAYS(${timestamp})"
-        + " - DAYS(CAST(#{now} as TIMESTAMP))) AND "
-        + "#{selectedItem.lowerAgeLimit} &lt;= (DAYS(${timestamp})"
-        + " - DAYS(CAST(#{now} as TIMESTAMP))) "
+        + "#{selectedItem.upperAgeLimit} >= (DAYS(${timestamp}) - DAYS(CAST(#{now} as TIMESTAMP)))"
+        + " AND "
+        + "#{selectedItem.lowerAgeLimit} &lt;= (DAYS(${timestamp}) - DAYS(CAST(#{now} as TIMESTAMP"
+        + "))) "
+        + "</if> "
+        + "<if test=\"_databaseId == 'oracle'\">"
+        + "#{selectedItem.upperAgeLimit} >= (TRUNC(${timestamp}) - "
+        + "TRUNC(CAST(#{now} as TIMESTAMP))) AND "
+        + "#{selectedItem.lowerAgeLimit} &lt;= (TRUNC(${timestamp}) - "
+        + "TRUNC(CAST(#{now} as TIMESTAMP))) "
```

Comment:
```suggestion
        + "<if test=\"_databaseId == 'oracle'\">"
        + "#{selectedItem.upperAgeLimit} >= (TRUNC(${timestamp}) - TRUNC(CAST(#{now} as TIMESTAMP)))"
        + " AND "
        + "#{selectedItem.lowerAgeLimit} &lt;= (TRUNC(${timestamp}) - TRUNC(CAST(#{now} as TIMESTAMP))) "
```

Reply:
not possible

[WARN] /home/arl/sources/taskana/taskana/lib/taskana-core/src/main/java/pro/taskana/monitor/internal/MonitorMapperSqlProvider.java:203: Line is longer than 100 characters (found 101). [LineLength]
[WARN] /home/arl/sources/taskana/taskana/lib/taskana-core/src/main/java/pro/taskana/monitor/internal/MonitorMapperSqlProvider.java:205: Line is longer than 100 characters (found 105). [LineLength]